src/Mobs/Enderman.cpp

```diff
@@ -149,6 +149,16 @@ void cEnderman::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 		return;
 	}
 
+	if (m_EMState != CHASING)
+	{
+		cAggressiveMonster* target = nullptr;
+		if (CanSeeMobType(mtEndermite, target, 64))
```

Comment:
```suggestion
		auto target = GetMonsterOfTypeInSight(mtEndermite, 64);
		if (target != nullptr)
```

Reply:
Why shouldn't you?

Reply:
Generally using `auto` is preferred, it makes later refactoring easier (say that the function changes the return type, you don't have to go through every usage of the function)